libs/api/domains/air-discount-scheme/src/lib/discount/discount.service.ts

```diff
@@ -39,6 +39,13 @@ export class DiscountService {
     return credit >= 1
   }
 
```

Comment:
In lint warnings above:
Maybe change to `Error` or `unknown`?

Reply:
:eyes:

Reply:
Changed the types to `(FetchError | ApolloError): void`